diff --git a/node_modules/@vanilla-extract/css/dist/vanilla-extract-css.esm.js b/node_modules/@vanilla-extract/css/dist/vanilla-extract-css.esm.js
index 1e5c118..7e11873 100644
--- a/node_modules/@vanilla-extract/css/dist/vanilla-extract-css.esm.js
+++ b/node_modules/@vanilla-extract/css/dist/vanilla-extract-css.esm.js
@@ -6,7 +6,6 @@ import { getAndIncrementRefCounter, getFileScope, hasFileScope } from '../fileSc
 import { walkObject, get } from '@vanilla-extract/private';
 import cssesc from 'cssesc';
 import { diff } from 'deep-object-diff';
-import chalk from 'chalk';
 import { _ as _taggedTemplateLiteral } from './taggedTemplateLiteral-b4c22b04.esm.js';
 import outdent from 'outdent';
 import deepmerge from 'deepmerge';
@@ -191,13 +190,13 @@ function diffLine(value, nesting, type) {
   var whitespace = [...Array(nesting).keys()].map(() => '  ').join('');
   var line = "".concat(type ? type : ' ').concat(whitespace).concat(value);
 
-  if (process.env.NODE_ENV !== 'test') {
+  if (process.env.NODE_ENV === 'development') {
     if (type === '-') {
-      return chalk.red(line);
+      return require('chalk').red(line);
     }
 
     if (type === '+') {
-      return chalk.green(line);
+      return require('chalk').green(line);
     }
   }
 
