diff --git a/node_modules/solid-start-cloudflare-pages/index.js b/node_modules/solid-start-cloudflare-pages/index.js
index f7e155f..3e1054d 100644
--- a/node_modules/solid-start-cloudflare-pages/index.js
+++ b/node_modules/solid-start-cloudflare-pages/index.js
@@ -60,9 +60,9 @@ export default function () {
           json(),
           nodeResolve({
             preferBuiltins: true,
-            exportConditions: ["node", "solid"]
+            exportConditions: ["node", "solid"],
           }),
-          common()
+          common(),
         ]
       });
 
